packages/tailwindcss-language-server/src/project-locator.test.ts

```diff
@@ -124,11 +124,15 @@ testFixture('v4/workspaces', [
       '{URL}/packages/admin/app.css',
       '{URL}/packages/admin/package.json',
       '{URL}/packages/admin/tw.css',
+      '{URL}/packages/shared/**',
+      '{URL}/packages/shared/ui.css',
     ],
   },
   {
     config: 'packages/web/app.css',
     selectors: [
+      '{URL}/packages/shared/**',
+      '{URL}/packages/shared/ui.css',
```

Comment:
I think these showing up indicates a problem with this approach. These show up because this is an imported file. *this is generally fine*

However, in this case it might be unexpected. Especially if `packages/shared` has its own Tailwind CSS config. It'll prefer packages/admin over packages/shared and that feels like a bug. 🤔

Reply:
I figured out a temporary solution that doesn't introduce potential issues like this. But it's kinda awful.

Will ship with it but it could use some improvement later.